templates/default/postgres.yaml.erb

```diff
@@ -1,53 +1,21 @@
-init_config:
-# empty by design
+# Generated by Chef, local modifications will be overwritten
+
+<%# TODO: Breaking change, remove defaults -%>
```

Comment:
It's a bummer the `datadog_monitor` LWRP doesn't provide a mechanism for passing the raw file content in to the LWRP. Over time I've tried to avoid doing logic like this in the template file, because they can get pretty intense pretty quick.

I wonder if there is any interest in supporting that within the LWRP itself.